instances/infrastructure-committee.near/widget/components/molecule/SimpleMDE.jsx

```diff
@@ -159,7 +160,13 @@ function getSuggestedAccounts(term) {
   let results = [];
 
   term = (term || "").replace(/\W/g, "").toLowerCase();
-  const limit = 5;
+   let limit = 5;
+  if (term.length < 2) {
+   results = [${sortedRelevantUsers
+     .map((u) => "{accountId:'" + u + "', score: 60}")
+     .join(",")}];
+    limit = ${5 + sortedRelevantUsers.length};
+  }
```

Comment:
I think `SimpleMDE` is pretty much same for all three instances, we can reuse it. wdyt?

Reply:
Good idea, I had a seperate branch that replaced a lot of components but I can do it in this PR for sure.